### ISSUE TITLE
Cannot use DataFormat (enum Turtle, RDFXML, ...) with @okp4/cognitarium-schema

### ISSUE THREAD

**Issue by antho31** [okp4/okp4-contract-schema#9]
Here is an enum declaration we can find in the `schema.d.ts` file from [@okp4/cogniatrium-schema](https://www.npmjs.com/package/@okp4/cognitarium-schema): 

```typescript
export declare enum RDFXML {
    NQuads = "n_quads",
    NTriples = "n_triples",
    RDFXML = "rdf_xml",
    Turtle = "turtle"
}
```

When I import `RDFXML` within a frontend (`import { RDFXML } from "@okp4/cognitarium-schema";`), I got the following message: 

`SyntaxError: The requested module '/node_modules/.vite/deps/@okp4_cognitarium-schema.js?t=1695034170441&v=b1b30273' does not provide an export named 'RDFXML'`
 
I am trying to figure out how I can use it. All the other imports work perfectly. 

Note that according to the contracts repo, [msg.rs from okp4-cognitarium](https://github.com/okp4/contracts/blob/main/contracts/okp4-cognitarium/src/msg.rs) declare `DataFormat` instead of `RDFXML`. Strange renaming, don't know if it's related.

Quicktype Issue submitted by @bdeneux: https://github.com/glideapps/quicktype/issues/2410

FYI @amimart @ccamel

**Comment by ccamel** [comment]
@bdeneux What's your analysis of this (rather old) issue?

**Comment by bdeneux** [comment]
@ccamel An issue has been reported on the QuickType repository ([#2410](https://github.com/glideapps/quicktype/issues/2410)), and a response has been recently posted. However, there doesn't appear to be any further updates at this time.

I plan to investigate whether this issue also affects another language generation, specifically Go, and will provide feedback accordingly. 

We may need to consider exploring other libraries that are more specifically designed for handling JSON schemas in smart contracts.

**Comment by bdeneux** [comment]
For TypeScript generation, we could potentially use this library, which is specifically designed for CosmWasm smart contracts: https://github.com/CosmWasm/ts-codegen. The major disadvantage is that this library does not support code generation for other languages.

**Comment by ccamel** [comment]
@bdeneux Yes, great idea! It's a lib that we had already identified, but without going any further. To be assessed. 🤓
